testing/run-scripts/run_cloud.sh

```diff
@@ -14,39 +14,42 @@ PREBUILT=
 NPM=true
 INVITE_CODE=
 REFRESH=false
+UPDATE=false
 PUBLIC_IP=
 BANNER=
 
 SSHD_PORT=5000
 
 function usage () {
-  echo "$0 [-p path] [-i invite code] [-r] [-d ip] [-b banner] browser-version"
-  echo "  -p: path to pre-built uproxy-lib repository"
-  echo "  -i: invite code"
-  echo "  -r: recreate Docker images (WARNING: will invalidate invite codes)"
-  echo "  -d: override the detected public IP (for development only)"
-  echo "  -b: name to use in contacts list"
-  echo "  -h, -?: this help message"
-  echo
-  echo "Example browser-version: chrome-stable, firefox-canary"
-  exit 1
+    echo "$0 [-p path] [-i invite code] [-r] [-u] [-d ip] [-b banner] browser-version"
```

Comment:
I like `-u`. I'm going to suggest we replace `-r` with `-u` and, in a followup, add a new switch which means "do not try to migrate data from any existing container". It will be a kind of "wipe everything" flag. Make sense?